x-pack/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/tabs/custom_urls.js

```diff
@@ -113,12 +122,16 @@ export class CustomUrls extends Component {
       })
       .catch((resp) => {
         console.log('Error building custom URL from settings:', resp);
-        toastNotifications.addDanger('An error occurred building the new custom URL from the supplied settings');
+        toastNotifications.addDanger(this.props.intl.formatMessage({
+          id: 'xpack.ml.jobsList.editJobFlyout.customUrls.addNewCustomUrlErrorNotificationMessage',
```

Comment:
```suggestion
          id: 'xpack.ml.jobsList.editJobFlyout.customUrls.addNewUrlErrorNotificationMessage',
```
I think `Custom` is redundant here, there is already `customUrls` segment.